tsconfig.json

```diff
@@ -17,6 +17,7 @@
 		"emitDecoratorMetadata": true
 	},
 	"include": [
-		"lib"
+		"lib",
+		"examples"
```

Comment:
AFAIK that would cause examples to be compiled and distributed with the NPM package... I think examples don't need to be published, just the actual library code

Reply:
Right, probably not sensible for a library module. But some instructions on building the examples would be good, also the ability to build them without installing. Guess I don't understand why they import from ".." if there's no code there?

Reply:
they import from '..' because that's the module itself :) so according to `package.json`, that resolves to `dist/index.js`

as per running / building examples, you're just supposed to run them with `ts-node` rather than building. I'll put a README in the examples folder

Reply:
Excellent idea :) thanks!